Automattic/jetpack
# Admin Page: disaplay notices when activating the Akismet plugin

#### Changes proposed in this Pull Request:

* This change gives a bit more feedback to site owners who click the link to activate the Akismet plugin.

![Screen Recording 2020-08-17 at 07 07 PM](https://user-images.githubusercontent.com/426388/90423863-84d1bc80-e0bd-11ea-81bd-7942ea76f8c0.gif)

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Edit `_inc/lib/class.core-rest-api-endpoints.php`, line 3728, from `if ( in_array( $status, array( 'active', 'network-active' ), true ) ) {` to `if ( in_array( $status, array( 'inactive', 'active', 'network-active' ), true ) ) {`.
* Install Akismet on your site, but do not activate it.
* Connect Jetpack to WordPress.com, pick a free plan.
* Go to Jetpack > Dashboard.
* Click to activate Akismet
* You should see something like the above.

#### Proposed changelog entry for your changes:

* N/A
